backend/server.py

```
@@ -1512,6 +1512,13 @@ async def lifespan(app_: "FastAPI"):
     except Exception as exc:
         log.warning("Startup schedule cleanup failed (non-fatal): %s", exc)
 
+    # One-shot boot purge (PURGE_BACKLOG_ON_BOOT nonce) — background task so
+    # draining a large Mongo backlog never blocks startup or the health check.
+    try:
+        asyncio.create_task(_maybe_boot_purge())
```

Ruff (0.15.20)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)